src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -338,3 +341,12 @@ func (updated *Config) updateFrom(updateBytes []byte) error {
 	}
 	return nil
 }
+
+func inHomeDir(filename string) string {
+	log.Tracef("Determining user's home directory")
+	usr, err := user.Current()
+	if err != nil {
+		panic(fmt.Errorf("Unable to determine user's home directory: %s", err))
```

Comment:
And also I guess the benefit to log.Fatalf is that the output looks less messy than the stacktrace from a panic.